src/services/dynamodb-service.ts

```diff
@@ -0,0 +1,29 @@
+import { DynamoDBClient, ListTablesCommand } from "@aws-sdk/client-dynamodb";
+import { HTTPChecker, IntegrationConfig } from "../interfaces/types";
+
+export async function checkDynamodbClient(config: IntegrationConfig): Promise<HTTPChecker> {
+  return new Promise(async (resolve, _) => {
```

Comment:
If you are not using the second parameter, you can simple remove them instead of use `_`

```suggestion
  return new Promise(async resolve => {
```